src/main/java/core/basesyntax/RandomSize.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomSize {

```

Comment:
Consider renaming the class to a more descriptive name such as 'RandomFigureSize'. 'RandomSize' is a bit generic and does not describe what kind of sizes are generated.

Reply:
Done it.